frontend/src/components/Navbars/TCNavbar.js

```diff
@@ -74,7 +75,8 @@ class TCNavbar extends React.Component {
                 <div className="navbar-collapse-header">
                   <Row>
                     <Col className="collapse-brand" xs="6">
-                      <Link to="/">
+                    
+                      <Link to="http://localhost:3000/">
```

Comment:
If you want, this was recently added as an environmental variable (REACT_APP_HOME_URL). If you rebase you should be able to access that change